opaque_keys/edx/keys.py

```diff
@@ -77,6 +77,26 @@ def block_type(self):  # pragma: no cover
         raise NotImplementedError()
 
 
+class AggregateCourseKey(OpaqueKey):
+    """
+    An :class:`opaque_keys.OpaqueKey` identifying a
+    serialized Course Key object.
+    """
+    KEY_TYPE = 'aggregate_course_key'
```

Comment:
I would call this course-v2 now. Add a ticket to your backlog to add course-run-v1.